code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/notification/PushTemplateImageHelper.kt

```diff
@@ -0,0 +1,284 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui.notification
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.widget.RemoteViews
+import com.adobe.marketing.mobile.core.R
+import com.adobe.marketing.mobile.services.Log
+import com.adobe.marketing.mobile.services.caching.CacheService
+
+internal object PushTemplateImageHelper {
```

Comment:
Now that we have individual objects for each of the builders, can you move the image population to their own builders.